### PR TITLE
ENH: Extend developer documentation

### PR DESCRIPTION
**Pull request by wgershwitz**
Related to #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,25 +8,117 @@
...
+        clusterrole.rbac.authorization.k8s.io/kubestone-proxy-role created
+        rolebinding.rbac.authorization.k8s.io/kubestone-leader-election-rolebinding created
+        clusterrolebinding.rbac.authorization.k8s.io/kubestone-manager-rolebinding created
+        clusterrolebinding.rbac.authorization.k8s.io/kubestone-proxy-rolebinding created
+        service/kubestone-controller-manager-metrics-service created
+        deployment.apps/kubestone-controller-manager created
+    ```
+
+### Run benchmark
+ * Create dedicated namespace for bencmarking
```
Comment (apollolemonte) [inline README.md]: benchmarking

File: README.md [hunk 2]
```diff
@@ -8,25 +8,117 @@
 [![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
 #
-kubestone is a benchmark oriented Kubernetes Operator. 
+kubestone is a benchmarking Kubernetes Operator.
```
Comment (apollolemonte) [inline README.md]: K for Kubestone?

File: README.md [hunk 3]
```diff
@@ -8,25 +8,117 @@
...
+for Kubernetes cluster via common set of benchmarks:
+ * Disk: [fio](https://fio.readthedocs.io)
+ * Network: [IPerf3](https://iperf.fr)
+ * CPU: [sysbench](https://wiki.gentoo.org/wiki/Sysbench)
  
- 
-## Development status
-Under development / alpha.
+## Benchmark definitions
+Benchmark are initiated by creating Custom Resources in
```
Comment (apollolemonte) [inline README.md]: Benchmarks?

File: README.md [hunk 4]
```diff
@@ -8,25 +8,117 @@
...
+any namespace in Kubernetes. When a new Kubestone CR is created
+the benchmark's workflow is executed.
 
 ## Usage
+### Prerequisities
```
Comment (apollolemonte) [inline README.md]: Prerequisites

File: README.md (hunk 1 above)
Comment (wgershwitz) [reply README.md]: good catch, fixed!

File: README.md (hunk 2 above)
Comment (wgershwitz) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (wgershwitz) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (wgershwitz) [reply README.md]: ouch... fixed.

**Comment by wgershwitz** [comment]
Also fixed the indenting of code blocks.

**Pull request merged**